auth-server/knexfile.ts

```diff
@@ -27,26 +29,10 @@ const config = {
 };
 
 const env = process.env.NODE_ENV || 'development';
-const connection = config[String(env)];
+const connection = config[env];
 
 if (!connection || !connection.connection) {
 	throw new Error(`DB Connection not provided for env ${env}`);
 }
 
-module.exports = connection;
-
-// staging: {
-//   client: 'postgresql',
-//   connection: {
-//     database: 'my_db',
-//     user:     'username',
-//     password: 'password'
-//   },
-//   pool: {
-//     min: 2,
-//     max: 10
-//   },
-//   migrations: {
-//     tableName: 'knex_migrations'
-//   }
-// },
+export = connection;
```

Comment:
This is very weird and made me lose quite some time. anything else like `export default connection` or `export const connection..` was resulting in `knex` not being able to import the options because it was nester in a "connection" object, or "default" object that looked like:
```bash
{
  default: { // <-- this is wrong
    client:...
    ...
  }
}
``` 

where it should have been:
```bash
{
  client:...
  ...
}
```